required/latex-lab/testfiles-LM/LM-3-1+2.tlg

```diff
@@ -1,88 +1,88 @@
-Package tagpdf Info: closing structure 12 tagged /LBody
-Package tagpdf Info: closing structure 10 tagged /LI
-Package tagpdf Info: closing structure 9 tagged /itemize
-Package tagpdf Info: closing structure 8 tagged /text-unit
-Package tagpdf Info: closing structure 6 tagged /Sect
-Package tagpdf Info: The rule between parent 'Document' 
-(tagpdf)             and child 'Sect' is '1 (0..n)'
-Package tagpdf Info: The rule between parent 'Sect' 
-(tagpdf)             and child 'H2 (from section)' is '1 (0..n)'
-Package tagpdf Info: The rule between parent 'H2 (from section)' 
-(tagpdf)             and child 'Lbl' is '1 (0..n)'
-Package tagpdf Info: The rule between parent 'Lbl' 
-(tagpdf)             and child 'MC' is '1 (0..n)'
-Package tagpdf Info: closing structure 17 tagged /Lbl
-Package tagpdf Info: The rule between parent 'H2 (from section)' 
-(tagpdf)             and child 'MC' is '1 (0..n)'
-Package tagpdf Info: closing structure 16 tagged /section
-Package tagpdf Info: closing structure 15 tagged /Sect
-Package tagpdf Info: The rule between parent 'Document' 
-(tagpdf)             and child 'Sect' is '1 (0..n)'
-Package tagpdf Info: The rule between parent 'Sect' 
-(tagpdf)             and child 'H2 (from section)' is '1 (0..n)'
-Package tagpdf Info: The rule between parent 'H2 (from section)' 
-(tagpdf)             and child 'Lbl' is '1 (0..n)'
-Package tagpdf Info: The rule between parent 'Lbl' 
-(tagpdf)             and child 'MC' is '1 (0..n)'
-Package tagpdf Info: closing structure 20 tagged /Lbl
-Package tagpdf Info: The rule between parent 'H2 (from section)' 
-(tagpdf)             and child 'MC' is '1 (0..n)'
-Package tagpdf Info: closing structure 19 tagged /section
+Package tagpdf Info: Rule 'Document' --> 'Sect' is 1 (='0..n')
+(tagpdf)             Rolemapped from: 'Document' --> 'Sect' on line ...
+Package tagpdf Info: Rule 'Sect' --> 'H2' is 1 (='0..n')
+(tagpdf)             Rolemapped from: 'Sect' --> 'section' on line ...
+Package tagpdf Info: Rule 'H2' --> 'MC' is 1 (='0..n')
+(tagpdf)             Rolemapped from: 'H2' --> 'MC' on line ...
+Package tagpdf Info: closing structure 9 tagged /section
+Package tagpdf Info: Rule 'Sect' --> 'Part' is 1 (='0..n')
+(tagpdf)             Rolemapped from: 'Sect' --> 'text-unit' on line ...
+Package tagpdf Info: Rule 'Sect' --> 'L' is 1 (='0..n')
+(tagpdf)             Rolemapped from: 'Sect' --> 'itemize' on line ...
+Package tagpdf Info: Rule 'L' --> 'LI' is 1 (='0..n')
+(tagpdf)             Rolemapped from: 'L' --> 'LI' on line ...
+Package tagpdf Info: Rule 'LI' --> 'Lbl' is 1 (='0..n')
+(tagpdf)             Rolemapped from: 'LI' --> 'Lbl' on line ...
+Package tagpdf Info: Rule 'Lbl' --> 'MC' is 1 (='0..n')
+(tagpdf)             Rolemapped from: 'Lbl' --> 'MC' on line ...
+Package tagpdf Info: closing structure 13 tagged /Lbl
+Package tagpdf Info: Rule 'LI' --> 'LBody' is 1 (='0..n')
+(tagpdf)             Rolemapped from: 'LI' --> 'LBody' on line ...
```

Comment:
I must confess I don't don't understand the "Rollmapped from" lines.  To me this looks as if LI is turned to LBody (via role mapping and that makes no sense to me)